client/caps.go

```diff
@@ -83,3 +83,19 @@ func (client *Client) AddCapability(c *Capability) error {
 	}
 	return nil
 }
+
+// RemoveCapability removes one capability from the system
+func (client *Client) RemoveCapability(name string) error {
+	errPrefix := "cannot remove capability"
+	var rsp response
+	if err := client.do("DELETE", fmt.Sprintf("/1.0/capabilities/%s", name), nil, &rsp); err != nil {
```

Comment:
Do we need to be concerned about quoting here? I assume the anser is no, but I would like to be confirmed